lib/SILOptimizer/SemanticARC/OwnershipLiveRange.h

```diff
@@ -56,6 +52,10 @@ class LLVM_LIBRARY_VISIBILITY OwnershipLiveRange {
   /// the uses.
   SmallVector<Operand *, 6> consumingUses;
 
+  /// The value that we are computing the LiveRange for. Expected to be an owned
+  /// introducer and not to be forwarding.
+  OwnedValueIntroducer introducer;
```

Comment:
This just makes the small vector of consuming uses first to pack the data structure a little better (I am going to in a future commit see what consuming uses should actually be looking at data, but I wanted at the time when I converted from storing consuming uses separately to storing them together to match the total size (it was SmallVector 2 for unknown consuming use, destroying use, and forwarding uses, so 6.